charts/bungee/templates/deployment.yaml

```
@@ -0,0 +1,187 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "bungee.fullname" . }}
+  {{- if .Values.deploymentAnnotations }}
+  annotations:
+    {{- range $key, $value := .Values.deploymentAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+  {{- end }}
```

I'm not sure which approach is better, but for consistency could use
```suggestion
{{ toYaml .Values.deploymentAnnotations | indent 4 }}
```